src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -1001,30 +987,87 @@ private static PlatformAttributes CopyAllAttributes(PlatformAttributes copyTo, P
                     attributes = CopyAttributes(containerAttributes);
                 }
 
-                AddPlatformAttributes(symbol.GetAttributes(), ref attributes);
-
-                if (symbol is IMethodSymbol method && method.IsAccessorMethod())
-                {
-                    // Add attributes for the associated Property
-                    AddPlatformAttributes(method.AssociatedSymbol.GetAttributes(), ref attributes);
-                }
+                MergePlatformAttributes(symbol.GetAttributes(), ref attributes);
 
                 attributes = platformSpecificMembers.GetOrAdd(symbol, attributes);
             }
 
             return attributes != null;
 
-            static bool AddPlatformAttributes(ImmutableArray<AttributeData> immediateAttributes, [NotNullWhen(true)] ref SmallDictionary<string, PlatformAttributes>? attributes)
+            static void MergePlatformAttributes(ImmutableArray<AttributeData> immediateAttributes, [NotNullWhen(true)] ref SmallDictionary<string, PlatformAttributes>? parentAttributes)
```

Comment:
There is no return value now.
```suggestion
            static void MergePlatformAttributes(ImmutableArray<AttributeData> immediateAttributes, ref SmallDictionary<string, PlatformAttributes>? parentAttributes)
```